usaspending_api/recipient/v2/views/recipients.py

```diff
@@ -392,6 +394,20 @@ def get(self, request, recipient_id):
         return Response(result)
 
 
+@method_decorator(deprecated, name="get")
+class RecipientOverViewDuns(RecipientOverView):
+    """
+    <em>Deprecated: Please see <a href="../recipient/recipient_id">this endpoint</a> instead.</em>
```

Comment:
Will need to update this message to use a Recipient Hash in the URL since that is a parameter value. Could use the currently referenced recipient hash in `endpoints.md` of `99a44eeb-23ef-e7c4-1f84-9a695b6f5d2e-R`.